cicd/_common_deploy_logic.sh

```diff
@@ -90,14 +92,14 @@ function teardown {
         echo "Running teardown for ns: $ns"
         set +e
 
-        if [ "$CAPTURED_SIGNAL" == "EXIT" ] && check_junit_files "${ARTIFACTS_DIR}/junit-*.xml"; then
+        if [ "$ALWAYS_COLLECT_LOGS" != "true" ] && [ "$CAPTURED_SIGNAL" == "EXIT" ] && check_junit_files "${ARTIFACTS_DIR}/junit-*.xml"; then
             echo "No errors or failures detected on JUnit reports, skipping K8s artifacts collection"
         else
-            echo "Errors or failures detected, collecting K8s artifacts"
+            [ "$ALWAYS_COLLECT_LOGS" != "true" ] && echo "Errors or failures detected, collecting K8s artifacts"
```

Comment:
nitpick: the message is a bit confusing since you're actually forcing the log collection so not necessarily caused by an error or failure.  I thought of implement this with a small `if` clause for different messaging. Still splitting hairs so your call

Reply:
LOL - I don't know how to read - ignore me.